examples/ads.amp.html

```diff
@@ -349,6 +350,12 @@ <h2>AdOcean</h2>
           >
   </amp-ad>
 
+  <h2>AdPicker</h2>
```

Comment:
I tried to test with your ad. But it returns a no content ad to me. Could you provide a fake/demo ad example here for us to test. Thanks.